src/ex07.cairo

```diff
@@ -5,8 +5,13 @@
 // - Use this contract's claim_points() function
 // - Your points are credited by the contract
 ////////////////////////////////
+#[starknet::interface] 
+trait Ex07Trait<T> {
+    fn claim_points(value_a: u128, value_b: u128);
+    fn update_class_hash(class_hash: felt252);
+}
 
-#[contract]
+#[starknet::contract] 
```

Comment:
This contract does not have a storage value, ContractState cannot be accessed without this. If there are no storage values for this example I believe an empty storage value will suffice.

https://book.starknet.io/chapter_2/storage.html

Reply:
Thanks for the correction, I've implemented an empty storage value.